Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Payment.hs

```diff
@@ -108,7 +109,7 @@ getStatus (personId, merchantId) orderId = do
         processMandateStatus mandateStatus mandateId
       DPayment.PaymentStatus _ -> do
         processPayment merchantId (cast order.personId) order.id True
-  notifyIfPaymentFailed personId order.id order.status
+  notifyAndUpdateInvoiceStatusIfPaymentFailed personId order.id order.status
```

Comment:
`orderId` is not `invoiceId`. It is `driverFee` id, right?

Reply:
It is `invoiceId` now, according to new flow.